store/boltdb/boltdb.go

```diff
@@ -80,13 +81,15 @@ func New(endpoints []string, options *store.Config) (store.Store, error) {
 		if err != nil {
 			return nil, err
 		}
+	} else if options.ConnectionTimeout != 0 {
+		timeout = options.ConnectionTimeout
```

Comment:
Lets make this unconditional because the `options.ConnectionTimeout` is used always. This way BoltDB.timeout will reflect the right value if at all it has to be used elsewhere.